### DISCUSSION TITLE
alternatives to raspi to run mixxx (well)?

### DISCUSSION THREAD

**Discussion by inktrap** [fayaaz/mixxx-pi-gen#63] [General]
Given that this is a distro to use mixxx on a raspi with a hardware dj controller I guess it makes sense to ask this question here.

Which alternative raspberry pi would you recommend nowadays? The raspis are hard to get and the mixxx wiki and forum doesn't make it easy to find the recommended hardware specs … how much cpu/ram/aso is needed/ideal?

- Orange Pi 4
- Tinker Board S
- [Odroid XU4Q](https://magazine.odroid.com/article/odroid-xu4-review-better-performance-than-the-raspberry-pi-plus-usb3-and-gigabit-ethernet-for-usd60/)(2GB CPU, 2GB Ram): https://de.aliexpress.com/item/1005002035278133.html
- Orange Pi 5: https://aliexpress.com/item/1005004952870650.html

**Comment by fayaaz** [comment]
The problem with asking '...how much cpu/ram/aso is needed/ideal?' is that the answer is it depends!

Are you using keylock? Are your files all 128KB/s MP3 or FLAC with 96k sample rate? Do you need to use 4 decks? How much latency is acceptable? The answers to this will determine what level of hardware you will need.

Unfortunately I have no experience with the other SBCs. I'm not even sure if this image will boot on them (if they can boot the Raspberry Pi stock debian image then they should be able to boot this). The Pi is (was) ubiquitous, cheap and easy to target and probably importantly I had a few to hand to test the image on.

Honestly the speed was less than ideal for me even with a Raspberry Pi 4 to use this as a daily driver and I run Mixxx on a Steam Deck nowadays.

**Comment by inktrap** [comment]
Hey :) Yes, that point is very interesting: the failure-rate depends on decks, settings, locks, bitrate, format aso. If that rate is acceptable depends on the venue and the people.

Alright, my current plan is the following: get a used ddj400 pretty cheap and learn how to use it. Then it would be parties with friends and relaxed people who don't mind small occasional problems … either at home or at venues where I can't turn up the volume a lot. 

So these kinds of sessions, at home, at smaller alternative venues, or with friends/other people are an ideal opportunity to learn and I won't be using this in a club. It is functionality and skills first, mobility and the thoughts about reliability that come with that, second.

And I guess I would want to upcycle an older laptop of mine so it is capable to do the job, and if that doesn't work I'll invest in an Orange Pi. But that is like the third or fourth step and given my circumstances this makes sense.
